src/components/axes/timeAxis.ts

```diff
@@ -205,6 +208,7 @@ export module Axis {
         return this._possibleTimeAxisConfigurations;
       }
       this._possibleTimeAxisConfigurations = configurations;
+      this._maximumTiers = _Util.Methods.max(configurations.map((config: TimeAxisConfiguration) => config.length), 0);
```

Comment:
I guess I'm still a bit confused why this instance variable must be set here instead of calculating this value at the usage down below in _computeHeight